test/e2e/features/basic.feature

```diff
@@ -54,7 +54,7 @@ Feature: Basic test
         And stderr should contain "Checking if NetworkManager service is running"
         And stderr should contain "Using root access: Executing systemctl daemon-reload command"
         And stderr should contain "Using root access: Executing systemctl reload NetworkManager"
-        And stdout should contain "Your system is correctly setup for using CodeReady Containers, you can now run 'crc start' to start the OpenShift cluster"
+        And stdout should contain "Your system is correctly setup for using CodeReady Containers, run 'crc start' to start the instance"
```

Comment:
Still incorrect:

"Your system is correctly setup for using CodeReady Containers. Run 'crc start' to start the instance"